### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by sub1120**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+### Issue ticket number and link
+Write here...
+
+### Checklist before requesting a review
+- [ ] I have performed a self-review of my code
+- [ ] I have done code formating with prettier.
+- [ ] I have written explainable comments in the code. 
+- [ ] I have written conventional commit messages (https://www.conventionalcommits.org/en/v1.0.0/).
+- [ ] I confirm that code is upto date with main branch.
+- [ ] I confirm that UI is rendered properly, without any unexpected error.
```
Comment (shivamvijaywargi) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Add UI should be fully responsive as well.

**Review by shivamvijaywargi — CHANGES_REQUESTED** [review]
Please review the suggested changes

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 1 above)
Comment (sub1120) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Done, check now

**Review by shivamvijaywargi — APPROVED** [review]
Nice

**Pull request merged**